### ISSUE TITLE
Feature Request: Allow custom images

### ISSUE THREAD

**Issue by fmarzocca** [mopidy/mopidy#1793]
It will be very useful to have a way to assign a custom image to those personal playlists or artists without image.

**Comment by kingosticks** [comment]
Do you mean for Mopidy to return a default image when the request object doesn't have one, and make that default image customisable. Or do you mean provide a way for users to set an arbitrary image for a given object? The latter being significantly more complicated.

**Comment by fmarzocca** [comment]
I mean the first option.



- Sent from my mobile phone. Sorry for typos -

Il lun 9 set 2019, 21:41 Nick Steel <notifications@github.com> ha scritto:

**Comment by kingosticks** [comment]
As it is, when Mopidy doesn't provide an image, most clients then take it upon themselves to have their own go at finding an image i.e. searching last.fm etc. Which results (in my opinion) a better end-user experience. Enabling this feature would stop the client from doing that search, unless it's updated to account for the default image. Out of interest, what is your specific user-case?

**Comment by fmarzocca** [comment]
I'm using Iris and it doesn't provide anymore Artists images, so I thought
it would be nice to upload user's custom images.


- Sent from my mobile phone. Sorry for typos -

Il mar 10 set 2019, 16:30 Nick Steel <notifications@github.com> ha scritto:

**Comment by kingosticks** [comment]
Ahh, I see from the Iris [changelog](https://github.com/jaedb/Iris/releases/tag/3.38.0) that last.fm no longer provide artist images. That's a shame and really devalues their service. I'm not sure if this should be part of Mopidy or implemented by each client as they see fit.

**Comment by fmarzocca** [comment]
Yes, of course, sorry. There was a typo... I meant the second. (I edited the answer).
